src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/CpsBulkFileRestoreCoordinationEvent.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+
+namespace NuGet.VisualStudio
+{
+    public class CpsBulkFileRestoreCoordinationEvent : TelemetryEvent
```

Comment:
There should be an entry in the csproj for this new file. Make sure to save all and commit changes.

Reply:
I'm still not seeing an entry in any csproj file for this.

Reply:
discussed offline